roms.sha1

```diff
@@ -0,0 +1,2 @@
+ea9bcae617fdf159b045185467ae58b2e4a48b9a  pokered.gbc
+d7037c83e1ae5b39bde3c30787637ba1d4c48ce2  pokeblue.gbc
```

Comment:
Obviously not your fault originally, but a double space is intended for text files. md5sum/sha1sum mark digests of binary files with `*` instead (pokecrystal has this right).
Also, can you replace the md5s in README.md with the sha1s?